src/ffi/miniz_oxide.rs

```diff
@@ -146,7 +146,7 @@ impl From<FlushCompress> for MZFlush {
 impl DeflateBackend for Deflate {
     fn make(level: Compression, zlib_header: bool, _window_bits: u8) -> Self {
         // Check in case the integer value changes at some point.
-        debug_assert!(level.level() <= 10);
+        debug_assert!(level.level() <= 9);
```

Comment:
Why is this change necessary? `miniz_oxide` supports level 10 according to its [docs](https://docs.rs/miniz_oxide/latest/miniz_oxide/deflate/enum.CompressionLevel.html).